target/linux/realtek/files-5.15/drivers/net/phy/rtl83xx-phy.c

```diff
@@ -3870,7 +3853,7 @@ static int rtl9300_serdes_probe(struct phy_device *phydev)
 
 	phydev_info(phydev, "Detected internal RTL9300 Serdes\n");
 
-	return rtl9300_configure_serdes(phydev);
+	return 0;
```

Comment:
No need to configure because it's done at a later point anyway?

Reply:
Yes. There is not really a point to configuration of serdes until we know what link mode we want. So it is now all deferred until we try to establish a link.